### PR TITLE
Svenstaro.Genact version 1.3.0

### PR DESCRIPTION
**Pull request by JasonWei512**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/130129)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 10:44:23 AM: 
> 2023-12-12 10:44:22.084 [FAIL] Installer failed security check. Url: https://github.com/svenstaro/genact/releases/download/v1.3.0/genact-1.3.0-x86_64-pc-windows-msvc.exe Result: 0x80004005

**Comment by stephengillie** [comment]
Hi @JasonWei512,

The package didn't pass a Defender scan. This might be a false positive and we can rescan tomorrow.

File: manifests/s/Svenstaro/Genact/1.3.0/Svenstaro.Genact.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Svenstaro.Genact
+PackageVersion: 1.3.0
+InstallerType: portable
+Commands:
+- genact
```
Comment (stephengillie) [inline manifests/s/Svenstaro/Genact/1.3.0/Svenstaro.Genact.installer.yaml:8]:
```suggestion
- genact
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — APPROVED** [review]

**Comment by Trenly** [comment]
@stephengillie - This Package is causing a case sensitive issue with the existing `svenstaro.miniserve` package

**Pull request merged**